sys/src/cmd/frontusb/disk/scsireq.c

```diff
@@ -273,14 +274,14 @@ SRread(ScsiReq *rp, void *buf, i32 nbytes)
 
 	/* device is a tape or something similar */
 	if (rp->sense[2] == Sd2filemark || rp->sense[2] == 0x08 ||
-	    rp->sense[2] & Sd2ili && n > 0)
+	    (rp->sense[2] & Sd2ili && n > 0))
 		rp->data.count = nbytes - n;
 	else
 		return -1;
 	n = rp->data.count;
 	if (!rp->readblock++ || debug)
 		fprint(2, "SRread: tape data count %ld%s\n", n,
-			(rp->sense[2] & Sd2ili? " with ILI": ""));
+			(rp->sense[2] & Sd2ili? " with IQLockLI": ""));
```

Comment:
Is this right? ILI -> IQLockLI?

Reply:
This is definitely not right, and I have no idea how it got here.